src/Symfony/Component/Filesystem/Tests/Fixtures/MockStream/MockStream.php

```diff
@@ -0,0 +1,226 @@
+<?php
+
+/**
+ * This file is part of the Symfony package.
+ * (c) Fabien Potencier <fabien@symfony.com>.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * This class is based on VariableStream from the PHP Manual, which is licenced
+ * under Creative Commons Attribution 3.0 Licence copyright (c) the PHP
+ * Documentation Group
+ *
+ * @url http://php.net/manual/en/stream.streamwrapper.example-1.php
+ * @url http://php.net/license/
+ * @url http://creativecommons.org/licenses/by/3.0/legalcode
+ */
+namespace MockStream;
+
+/**
+ * Mock stream class to be used with stream_wrapper_register.
+ *
+ * stream_wrapper_register('mock', 'MockStream\MockStream')
+ */
+class MockStream {
```

Comment:
`{` should be on a new line. Actually, the whole file should be fixed for coding stantards.